powertools-e2e-tests/src/test/java/software/amazon/lambda/powertools/testutils/tracing/SegmentDocument.java

```diff
@@ -0,0 +1,189 @@
+package software.amazon.lambda.powertools.testutils.tracing;
```

Comment:
Feels like this should be a class published in the SDK?

Reply:
Maybe yes...